core/tests/karma.conf.ts

```diff
@@ -104,16 +104,14 @@ module.exports = function(config) {
     singleRun: true,
     customLaunchers: {
       Chrome_Travis: {
-        // Karma can only connect to ChromeHeadless when inside Docker.
-        base: isDocker ? 'ChromeHeadless' : 'Chrome',
+        base: 'ChromeHeadless',
         // Discussion of the necessity of extra flags can be found here:
         // https://github.com/karma-runner/karma-chrome-launcher/issues/154
         // https://github.com/karma-runner/karma-chrome-launcher/issues/180
-        flags: isDocker ? [
```

Comment:
Should be safe to remove L5
https://github.com/oppia/oppia/blob/58f72c94c6a76fd8f65a9043d39da224fa1ff920/core/tests/karma.conf.ts#L5

Reply:
removed!